build/macos-arm64/cross-compilation.meson

```diff
@@ -0,0 +1,11 @@
+[binaries]
+pkgconfig = ['pkg-config']
+c = 'clang'
+cpp = 'clang++'
+objc = 'clang'
```

Comment:
Should this not be `clang++`?

Reply:
I'm unsure here - I only filled it in because meson was complaining that it needed an `objc` entry here in order to do the cross compile, even though none of the libraries have objc code. I understand that this is all the same binary anyway, so maybe it's OK but if you can find a doc that says that clang++ is recommended for this then I can change it

Reply:
I had to change this for the @ImageMagick project earlier but we can keep it like this if it works.